projectforum/projects/project_filters.py

```diff
@@ -93,15 +154,15 @@ def errorMessage(error="Apologies, we could not process the request you've"
     return HttpResponse(json.dumps(response), content_type='application/json')
 
 
-def check_length(start, end, projects):
-    """
-    Make sure the length is within confines.  Will throw an error if I can't
-    get any projects based on the limits, otherwise will make sure I don't
-    overflow.
-    """
-    length_projects = len(projects)
-    if length_projects < start:
-        return -1
-    if length_projects > end:
-        return end
-    return length_projects
+# def check_length(start, end, projects):
```

Comment:
I wanted to keep it until I wrote the pagination into the project/list page.  Just in case.  Overall though, 90% change I won't need it (will delete when I'm positive in my pagination branch).